### DISCUSSION TITLE
How do I change dark mode background color?

### DISCUSSION THREAD

**Discussion by auroraongithub** [lxndrblz/anatole#442] [Q&A]
Hi, I was able to set up my website and host it on github (https://blog.nijika.de/), but I want to change the dark mode background color. I tried messing with the colors in the module folder in themes folder, yet to no avail. I tried forking the repo and changed it there first before adding it as submodule, didn't work. I even tried to mess with the scss in the resources/_gen folders, but it didn't work. Please help lmao

**Comment by auroraongithub** [comment]
also another question if you don't mind, my favicon doesn't work for some reason, and idk how to fix that ;-;

**Comment by lxndrblz** [comment]
@auroraongithub  please see the documentation:

https://github.com/lxndrblz/anatole/wiki/%F0%9F%92%85-Appearance#prefering-the-dark-theme

All variables for the colors are defined in this file:
https://github.com/lxndrblz/anatole/blob/master/assets/scss/modules/_variables.scss

**Reply by auroraongithub** [reply]
hi thanks for the quick reply i was already sleeping, that is the file i edited in my local copy but didn't change anything when i used hugo server

**Reply by auroraongithub** [reply]
i also installed it as a submodule, so im not sure i can edit it since you own the repo
